### ISSUE TITLE
Iatrogenic anophthalmos

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#11146]
Preferred term label:
Iatrogenic anophthalmos

Synonyms
Surgical removal of the eye, Post-enucleation anophthalmos, Acquired anophthalmos

Definition (free text, please give PubMed ID)
A form of acquired anophthalmos resulting from surgical removal of the eye (enucleation or evisceration), typically performed due to trauma, intraocular tumors, intractable pain in a blind eye, or severe infection. Unlike congenital anophthalmos, the orbit may have previously developed normally and prosthetic rehabilitation is often pursued.

Parent term (use hpo.jax.org/app)
Abnormal eye morphology (HP:12372)

Diseases characterized by this term
Retinoblastoma, Ocular trauma, Painful blind eye

Your nano-attribution (ORCID)
0000-0001-9042-0508

**Comment by pnrobinson** [comment]
@daboe01 
Adding new term: 
- ID: HP:6001313
- label: Iatrogenic anophthalmos
- parents: [Abnormal eye morphology (HP:0012372)](https://hpo.jax.org/app/browse/term/HP:0012372)
- pmid: 
- def: A form of acquired anophthalmos resulting from surgical removal of the eye (enucleation or evisceration), typically performed due to trauma, intraocular tumors, intractable pain in a blind eye, or severe infection. Unlike congenital anophthalmos, the orbit may have previously developed normally and prosthetic rehabilitation is often pursued.
- comment: 
- orcid: orcid:0000-0001-9042-0508
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11146
- synonyms:
